.gitignore

```diff
@@ -9,3 +9,4 @@ package-lock.json
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+sendgrid.env
```

Comment:
fine by me using sendgrid if you're into it, but might be cleaner to use [amazon ses](https://aws.amazon.com/ses/) since we're already using the aws-sdk

Reply:
good idea